src/plugins/discover/public/application/main/components/layout/flyout_panels.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ExpandableFlyoutProps } from '@kbn/expandable-flyout';
+import React from 'react';
+import DiscoverGridFlyout, {
+  DiscoverGridFlyoutProps,
+} from '../../../../components/discover_grid_flyout/discover_grid_flyout';
+export const expandableFlyoutDocumentsPanels: ExpandableFlyoutProps['registeredPanels'] = [
```

Comment:
@davismcphee we could register panels in the discover panel via the extension points...but in a scenario where we wanted to render a whole different kind of flyout (i.e. security), visualization, or panel...This allows us to co-locate and manage different flyout types